syntax = "proto3";
package NKikimrConfig;
option java_package = "ru.yandex.kikimr.proto";

message TAllocatorConfig {
    map<string, string> Param = 1;
    bool EnableDefaultAllocator = 2;
};
